README.md

```diff
@@ -168,6 +169,10 @@ $ curl http://localhost:8080/micro/inventory
 
 That's it, you have successfully deployed and tested the Inventory microservice.
 
+## Deploy Inventory Application on OpenLiberty
+
+The Spring Boot applications can be deployed on WebSphere application server Liberty as well. When the application is deployed on Liberty, the default server that is embedded as part of the application will be disabled and it will use Liberty instead of it. For instructions on how to deploy it on OpenLiberty, follow the instructions [here](OpenLiberty.MD).
```

Comment:
"application server" should be capitalized, as it is a proper name - WebSphere Application Server.

Reply:
Changed it to `WebSphere Application Server`